chat.schildi.desktop.appdata.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally this should be maintained in your code repository instead of here.

Reply:
This is not with upstream. It'd be great to stay as close to upstream as possible.

Reply:
It's not, but it should be. It's something you can easily fix as the upstream of the fork.

Reply:
The problem is (same like Element, I assume) that it shouldn't clutter up other systems.

Reply:
It's not Flathub specific file. All other distributions, be it Ubuntu, Fedora, or even snap, benefit from it.